modules/atomic-widgets/assets/js/editor/div-block-view.js

```diff
@@ -101,10 +101,31 @@ const DivBlockView = BaseElementView.extend( {
 		};
 	},
 
+	getDroppableAxis() {
+		if ( this.isHorizontalAxis() ) {
+			return 'horizontal';
+		}
+
+		return 'vertical';
+	},
+
+	isHorizontalAxis() {
+		const styles = window.getComputedStyle( this.$el[ 0 ] );
+
+		if ( 'flex' === styles.getPropertyValue( 'display' ) ) {
```

Comment:
We prefer to avoid nested conditions.

```suggestion
		if ( 'flex' !== styles.getPropertyValue( 'display' ) ) {
			return false;
		}
```